idlj/src/main/java/com/sun/tools/corba/ee/idl/ValueEntry.java

```diff
@@ -71,19 +71,25 @@ public void generate (Hashtable symbolTable, PrintWriter stream)
   /** Add an InterfaceEntry to the list of interfaces which this value
       supports.  During parsing, the parameter to this method COULD be a
       ForwardEntry, but when parsing is complete, calling supports will
-      return a vector which only contains InterfaceEntry's. */
+    * return a vector which only contains InterfaceEntry's.
+    * @param supports a {@link ForwardEntry} or {@link InterfaceEntry}
+    */
   public void addSupport (com.sun.tools.corba.ee.idl.SymtabEntry supports)
   {
     _supports.addElement (supports);
   } // addSupport
 
-  /** This method returns a vector of InterfaceEntry's. */
+  /** This method returns a vector of InterfaceEntry's.
```

Comment:
Previously, you changed a plural that used an apostrophe. Here you're adding one that does. Maybe use the same style consistently?